### ISSUE TITLE
Have you got quality result on ADE20K?

### ISSUE THREAD

**Issue by JackHenry1992** [DrSleep/tensorflow-deeplab-resnet#112]
![image](https://user-images.githubusercontent.com/9390228/29394599-09e21ac2-833e-11e7-913b-b5e08358c704.png)
![image](https://user-images.githubusercontent.com/9390228/29394639-3b2b1c3c-833e-11e7-99ef-b8125544ebc8.png)
Seems that loss stay at 0.65. And indoor scene cannot got quality result.
Have you got quality result on ADE20K dataset? **Can you upload your pre-trained model on ADE20K.**
Thanks a lot for sharing your code.

**Comment by DrSleep** [comment]
have not trained it on ADE20K
